Gemfile

```diff
@@ -35,7 +35,6 @@ group :development, :test do
   gem 'niftany'
   gem 'rspec-rails'
   gem 'rubocop'
-  gem 'solr_wrapper', github: 'cbeer/solr_wrapper', branch: 'master'
```

Comment:
also solr_wrapper gem in development?

Reply:
Ack, I removed by accident, in my last commit. Originally it was moved to dev at ae3eaf41cfdde8397cd7f1042e7899dc58b048e3